Demo/VIper-v2-Demo/VIper-v2-Demo/Common/Models/JSONAPIObject.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  JSONAPIObject.swift
+//  Pokedex
+//
+//  Created by Filip Beć on 28/04/2017.
+//  Copyright © 2017 Filip Beć. All rights reserved.
+//
+
+import UIKit
+import Unbox
+
+protocol JSONAPIModel: Unboxable {
```

Comment:
We should maybe updating project to use `Codable`

Reply:
Yes, please definitely use Codable/Japx combination.